src/main/java/use_cases/create_curriculum/CreateCurriculumPresenter.java

```diff
@@ -9,13 +11,21 @@
  */
 public class CreateCurriculumPresenter implements CreateCurriculumOutputBoundary {
 
+    private final List<DashboardUiInterface> dashboardObservers;
+
+    public CreateCurriculumPresenter(List<DashboardUiInterface> dashboardObservers) {
```

Comment:
Implementing the list of DashboardUIInterface, which helps with the DisplayCurriculumModel. Observers view will be beneficial for other UI components of the assignment.

Reply:
The Dashboard UI will implement this new interface so that it knows when a new curriculum has successfully been created